woocommerce/woocommerce-ios
# Local notification: Notification for Free trial survey

Part of: #10266 

## Description

Schedules a notification 24 hours after the Free trial subscription. We will be displaying the survey screen in a future PR. 

## Testing instructions
- Build and run the app on a physical device for easy testing.
- Create a new store with a free trial subscription.
- When the store creation completes, notice in Xcode console: 
```
Tracked local_notification_scheduled, properties: [AnyHashable("is_wpcom_store"): true, AnyHashable("is_iap_available"): true, AnyHashable("type"): "free_trial_survey_24h_after_free_trial_subscribed", AnyHashable("blog_id"): 12345]
```
- Open the device Settings app to switch to 23h58m after the time of the store creation.
- After 2 minutes, a notification asking for purchasing a plan should be displayed.
- Tap the notification, the app should be opened. Notice in Xcode console: 
```
Tracked local_notification_tapped, properties: [AnyHashable("blog_id"): 12345, AnyHashable("is_iap_available"): true, AnyHashable("is_wpcom_store"): true, AnyHashable("type"): "free_trial_survey_24h_after_free_trial_subscribed"]
```

## Screenshots
<img src="https://github.com/woocommerce/woocommerce-ios/assets/524475/d67f2569-9176-48bb-a792-58ee4733c618" width="320"/>

---
